sdk/inc/azure/core/internal/az_mqtt_policy.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should the filename be `az_mqtt_policy_internal.h` too?

Reply:
Renamed, thanks for catching that